prov/efa/src/rxr/rxr_pkt_type_req.c

```diff
@@ -45,61 +45,52 @@
  * types.
  */
 struct rxr_req_inf {
-	uint64_t protover;
+	uint64_t extra_info_id;
 	uint64_t base_hdr_size;
 	uint64_t ex_feature_flag;
 };
 
-/*
- * starting from protocol version 4, each REQ packet type will be assigned a
- * version number, and once assigned, the version number will not change.
- *
- * Baseline features will always be version 4 features, baseline and
- * not have a ex_feature_flag.
- *
- * Each extra feature will be assign a version and an ex_feature_flag.
- * Each extra feature will correspond to 1 or more REQ packet types.
- */
 static const
 struct rxr_req_inf REQ_INF_LIST[] = {
 	/* rtm header */
```

Comment:
is `extra_info_id` always 0?

Reply:
No.

The No. 65th's extra feature's extra_info_id will be 1.
Similiarly, the No. 129th extra feature's extra_info_id will be 2.

Reply:
I see, so `extra_info_id` is the offset of the 64 bit feature flag in the `extra_info` array. When we have more than 64 features we need to use the second element in `extra_info` to check it, and so on.

Reply:
But in that case what will be the 65th feature bit look like? `BIT_ULL(65)` or `BIT_ULL(1)`?

Reply:
65th feature will has `extra_info_id` being 1, and flag being `BIT_ULL(0)`.

`BIT_ULL(65)` does not exist because the flag is 64 bits

Reply:
Thanks, that makes sense.